test/val/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@
 set(VAL_TEST_COMMON_SRCS
   ${CMAKE_CURRENT_SOURCE_DIR}/../test_fixture.h
   ${CMAKE_CURRENT_SOURCE_DIR}/../unit_spirv.h
+  ${CMAKE_CURRENT_SOURCE_DIR}/val_code_generator.h
```

Comment:
Where is the .cpp file listed in the CMakeLists file?

Reply:
Done